docs/troubleshooting.rst

```diff
@@ -0,0 +1,45 @@
+Troubleshooting
+===============
+
+Aqueduct in Kubernetes
```

Comment:
We need to focus user that not aqueduct causes OOM but user model, lets rename this header to 'OOM in your model' or something like that